src/common/types.ts

```diff
@@ -14,4 +17,8 @@ export interface HotTableProps extends Handsontable.DefaultSettings {
   settings?: Handsontable.DefaultSettings
 }
 
+export interface HotTableComponent<V, D, M, C, P> extends ThisTypedComponentOptionsWithRecordProps<Vue, HotTableData, HotTableMethods, {}, HotTableProps> {
```

Comment:
Generic types `V, D, M, C, P` are not used. Should be passed over to the type that we want to extend. Otherwise generics are hardcoded here?
```suggestion
export interface HotTableComponent<V extends Vue, D, M, C, P> extends ThisTypedComponentOptionsWithRecordProps<V, D, M, C, P> {
```